src/components/ClassList.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import {
+    Box,
+    Flex,
+    Badge,
+    Center,
+    AspectRatio,
+    Spacer,
+    Image,
+    List,
+    ListItem,
+    VStack,
+    Grid,
+    GridItem,
+} from "@chakra-ui/react";
+import type { ClassCardInfo } from "models/Class";
+export const ClassList: React.FC<{ classInfo: ClassCardInfo[] }> = ({
+    classInfo,
+}) => {
+    // time in minutes is minutes from 0:00, convert to something like "1:23 am"
+    function convertToAmPm(timeInMinutes: number) {
+        return (
+            (Math.floor(timeInMinutes / 60) % 12).toString() +
+            ":" +
+            ("0" + (timeInMinutes % 60)).slice(-2) +
+            (timeInMinutes >= 720 ? "pm" : "am")
+        );
+    }
+    //  from https://stackoverflow.com/questions/196972/convert-string-to-title-case-with-javascript
```

Comment:
@mahadzaryab1 Is there a standard you would prefer for code documentation for helper functions like these